src/test/java/org/ays/auth/controller/AysUserControllerTest.java

```diff
@@ -297,4 +306,195 @@ void givenInvalidId_whenIdNotValid_thenReturnValidationError(String invalidId) t
                 .findById(Mockito.anyString());
     }
 
+
+    @Test
+    void givenValidIdAndUserUpdateRequest_whenUserUpdated_thenReturnSuccess() throws Exception {
+
+        // Given
+        String mockId = AysRandomUtil.generateUUID();
+        String mockEmailAddress = "test@email.com";
+        AysUserUpdateRequest mockUpdateRequest = new AysUserUpdateRequestBuilder()
+                .withValidValues()
+                .withPhoneNumber(new AysPhoneNumberRequestBuilder().withValidValues().build())
+                .withEmailAddress(mockEmailAddress)
```

Comment:
Email adresini ve telefon numarasını `withValidValues()` metoduna yazmamız daha iyi olabilir
```suggestion
        AysUserUpdateRequest mockUpdateRequest = new AysUserUpdateRequestBuilder()
                .withValidValues()
```